pkg/resourcewatch/git/git_store.go

```diff
@@ -0,0 +1,325 @@
+package git
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"k8s.io/kube-openapi/pkg/util/sets"
+
+	"gopkg.in/src-d/go-git.v4"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/yaml"
+)
+
+type GitStorage struct {
+	repo *git.Repository
+	path string
+}
+
+type gitOperation int
+
+const (
+	gitOpAdded gitOperation = iota
+	gitOpModified
+	gitOpDeleted
+	gitOpError
+)
+
+// NewGitStorage returns the resource event handler capable of storing changes observed on resource
+// into a Git repository. Each change is stored as separate commit which means a full history of the
+// resource lifecycle is preserved.
+func NewGitStorage(path string) (*GitStorage, error) {
+	// If the repo does not exists, do git init
```

Comment:
I wasn't gonna comment on the nit but since there is a verify-deps issue exists -> exist

Reply:
Looks like this typo has been around since the code was added in 2020, but I'll fix it now we've seen it 👍